### PR TITLE
Use CAPM3RELEASEBRANCH from jjb instead of setting it based on API Version

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by Rozzii — APPROVED** [review]

File: lib/common.sh [hunk 1]
```diff
@@ -123,28 +123,27 @@ export CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
...
+  IPAMBRANCH="${IPAMBRANCH:-release-1.1}"
 else
+  CAPM3BRANCH="${CAPM3BRANCH:-main}"
   IPAMBRANCH="${IPAMBRANCH:-main}"
 fi
 
+export IPAMPATH="${IPAMPATH:-${M3PATH}/ip-address-manager}"
+export IPAM_BASE_URL="${IPAM_BASE_URL:-metal3-io/ip-address-manager}"
+export IPAMREPO="${IPAMREPO:-https://github.com/${IPAM_BASE_URL}}"
+
```
Comment (mboukhalfa) [inline lib/common.sh:140]: I do not see a reason to move these lines here

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**